protobuf-kafkaconnect-converter/src/test/java/com/amazonaws/services/schemaregistry/kafkaconnect/protobuf/fromconnectdata/ConnectDataToProtobufDataConverterTest.java

```diff
@@ -92,7 +92,7 @@ public void convert_ForTimeType_ConvertsSuccessfully() {
     public void convert_ForDecimalType_ConvertsSuccessfully() {
         final DynamicMessage decimalMessage = ToProtobufTestDataGenerator.getProtobufDecimalMessage();
         final Descriptors.FileDescriptor fileDescriptor = decimalMessage.getDescriptorForType().getFile();
-        final Schema decimalSchema = ToProtobufTestDataGenerator.getDecimalSchema("DecimalDataTest");
+        final Schema decimalSchema = ToProtobufTestDataGenerator.getDecimalSchema("decimalProtobufSchema");
```

Comment:
Why change this?

Reply:
This is due to the struct implementation that the schema name needs to be consistent - we are getting the filedescriptorproto by name instead by the first one.